File: ExamSoftTechnologies/IssueTracker_Skeletons/C#/IssueTracker/Models/Issue.cs
namespace IssueTracker.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Issue
    {
        // TODO:
        [Key]
        public int Id { get; set; }

        [Required]
        public string Title { get; set; }

        [Required]
        public string Content { get; set; }

        [Required]
        public int Priority  { get; set; }
    }
}

File: ExamSoftTechnologies/IssueTracker_Skeletons/JavaScript/models/issue.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    // TODO:

    const Issue = sequelize.define("Issue", {
        title: {
            type: Sequelize.TEXT,
            allowNull: false,
        },
        content: {
            type: Sequelize.TEXT,
            allowNull: false,
        },
        priority:{
            type: Sequelize.INTEGER,
            allowNull: false,
        }
    }, {
        timestamps: false
    });

    return Issue;

};